src/Tizen.Multimedia.Vision/MediaVision/InferencePoseLandmarkDetector.cs

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using InteropPLD = Interop.MediaVision.InferencePoseLandmarkDetection;
+
+namespace Tizen.Multimedia.Vision
+{
+    /// <summary>
+    /// Provides the ability to detect pose landmark.
+    /// </summary>
+    /// <feature>http://tizen.org/feature/vision.inference</feature>
+    /// <feature>http://tizen.org/feature/vision.inference.face</feature>
+    /// <since_tizen> 12 </since_tizen>
+    public class InferencePoseLandmarkDetector : IDisposable
+    {
+        private IntPtr _handle;
+        private bool _disposed;
+
+        /// <summary>Initializes a new instance of the <see cref="InferencePoseLandmarkDetector"/> class.</summary>
+        /// <exception cref="NotSupportedException">The required features are not supported.</exception>
+        /// <since_tizen> 12 </since_tizen>
+        public InferencePoseLandmarkDetector()
+        {
+            ValidationUtil.ValidateFeatureSupported(VisionFeatures.Inference);
+            ValidationUtil.ValidateFeatureSupported(VisionFeatures.InferenceFace);
+
+            InteropPLD.Create(out _handle).Validate("Failed to create inference pose landmark detector.");
+
+            try
+            {
+                InteropPLD.Configure(_handle).Validate("Failed to configure inference pose landmark detector.");
+                InteropPLD.Prepare(_handle).Validate("Failed to prepare inference pose landmark detector.");
+            }
+            catch (Exception e)
+            {
+                Log.Error(MediaVisionLog.Tag, e.ToString());
+                InteropPLD.Destroy(_handle);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Finalizes an instance of the InferencePoseLandmarkDetector class.
+        /// </summary>
+        ~InferencePoseLandmarkDetector()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Detects pose landmarks on the source image synchronously.
+        /// </summary>
+        /// <remarks>
+        /// <see cref="InferencePoseLandmarkDetectorResult.Points"/> can be empty, if there's no detected pose landmark.
+        /// </remarks>
+        /// <param name="source">The image data to detect pose landmark.</param>
+        /// <returns>The points of detected pose landmark.</returns>
```

Comment:
we are returning points and that's good, but IMO it will sound better to say "The coordinates of detected pose landmarks." it focuses on the meaning/purpose more. Same with facial landmarks. Don't need to change all of the occurrences in a class.

Reply:
I'll fix it.